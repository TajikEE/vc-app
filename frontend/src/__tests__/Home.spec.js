import React from "react";
import { create } from "react-test-renderer";

import renderer from 'react-test-renderer';
import { Link } from 'react-router-dom';
import { StaticRouter } from 'react-router'

function Button(props) {
    return <button>Signup</button>;
  }

describe("button component", () => {
  test("matches the snapshot", () => {
    const button = create(<Button />);
    expect(button.toJSON()).toMatchSnapshot();
  });
});

test('link matches snapshot', () => {
  const component = renderer.create(
    <StaticRouter >
      <Link to="/login" />
    </StaticRouter>
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});