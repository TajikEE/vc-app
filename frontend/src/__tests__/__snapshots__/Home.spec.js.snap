// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component Matches the snapshot 1`] = `
<button>
  Signup
</button>
`;

exports[`Link matches snapshot 1`] = `
<a
  href="/login"
  onClick={[Function]}
/>
`;

exports[`button component matches the snapshot 1`] = `
<button>
  Signup
</button>
`;

exports[`link matches snapshot 1`] = `
<a
  href="/login"
  onClick={[Function]}
/>
`;
